### ISSUE TITLE
Accessibility fixes for Google search results route

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#9499]
From https://github.com/uiowa/uiowa/issues/9462

Color contrast does not meet minimum requirement, Role not inside the required context

On https://default.uiowa.ddev.site/search there might be a false positive when Siteimprove scans this page. "Custom search" shows up as orange text on white. Not sure if we can do much here as I believe this markup is furnished by Google.

This is not visible (parent wrapper is display:none), but Siteimprove sees it.

<img width="339" height="220" alt="Image" src="https://github.com/user-attachments/assets/324a8385-6675-43c5-b9f0-1e3abd2299cd" />

**Comment by bspeare** [comment]
@joewhitsitt could you provide more screenshots/instructions on how to reproduce this issue?

**Comment by bspeare** [comment]
Thanks @joewhitsitt.   I only get the "Sort by" to show up.  Do you see this on https://sandbox.prod.drupal.uiowa.edu/search?terms=Joe?  <img width="1383" height="582" alt="Image" src="https://github.com/user-attachments/assets/3e9f4f9d-22d9-4448-9959-e8f0f773df92" />

**Comment by bspeare** [comment]
For "Role not inside the required context" issue, as @joewhitsitt mentioned previously, we don't have access to that markup.   If we really want to remove it we could try with JS, but we don't have visibility when Google updates this markup.
